src/cmds/openapi.js

```diff
@@ -118,17 +118,10 @@ exports.run = async function (opts) {
 
     let bundledSpec;
     const oas = new OASNormalize(specPath, { colorizeErrors: true, enablePaths: true });
-    await oas.validate(false).catch(err => {
-      return Promise.reject(err);
+    await oas.validate(false);
+    await oas.bundle().then(res => {
+      bundledSpec = JSON.stringify(res);
     });
-    await oas
-      .bundle()
-      .then(res => {
-        bundledSpec = JSON.stringify(res);
-      })
-      .catch(err => {
-        return Promise.reject(err);
-      });
```

Comment:
All the changes in this file are unrelated to the fix, these are just minor refactors. These `.catch(err => Promise.reject(err));` statements are redundant since we'd be rejecting the promise and handling that in `bin/rdme` anyways.